### DISCUSSION TITLE
How do I connect Lem 2.0 to Sly?

### DISCUSSION THREAD

**Discussion by aykaramba** [lem-project/lem#733] [General]
I got some amazing help to build a working Lem 2.0 iage so I can run it on debian in this ticket:  https://github.com/lem-project/lem/issues/729

Lem sdl starts up so fast I really want to start using it more and more.  I cannot find information on how to connect it to sly on my system.  Does anyone know of any documentation for how this is done?

Thanks.

**Comment by cxxxr** [comment]
I am glad you used lem, Thank you.

probably you can start up slynk in the lem process and connect it with sly

launch lem with slynk
```
sbcl --eval '(ql:quickload :slynk)' --eval '(slynk:create-server :dont-close t)' --eval '(ql:quickload :lem-sdl2)' --eval '(lem:lem)'
```

emacs side
```
M-x sly-connect
```

**Reply by aykaramba** [reply]
Aha, interesting, let me play with that.  Thanks!

**Reply by Sasanidas** [reply]
If you want to launch slynk inside Lem, I have this command (https://gitlab.com/sasanidas/lem-config/-/blob/master/init.lisp#L53) in my config that maybe useful :+1:

**Reply by aykaramba** [reply]
Aha!  That works, thank you.
